documentation/src/main/asciidoc/topics/streams.adoc

```diff
@@ -214,7 +212,7 @@ payload size even further.
 
 [source,java]
 ----
-   Map<Object, String> jbossValues = cache.entrySet().stream()
+Map<Object, String> map = (Map<Object, String>) cache.entrySet().stream()
               .filter(new ContainsFilter("Jboss"))
               .collect(ToMapCollectorSupplier.INSTANCE);
```

Comment:
this collector doesn't exist.
I think you can use `.collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));`

Reply:
fixed. thanks @pruivo